multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CreateServiceStep.java

```diff
@@ -44,7 +42,10 @@ protected OperationExecutionState executeOperation(ProcessContext context, Cloud
             if (operationExecutionState.isPresent()) {
                 return operationExecutionState.get();
             }
-            processServiceCreationFailure(context, serviceInstance, e);
+
+            processServiceActionFailure(context, serviceInstance,
```

Comment:
If you're making the exception at the same time as calling that method - take the exception out of the method invocation on the line just above to make it more readable. e.g.:
  CloudOperationException createFailedException = createCloudOperationException(...);
  processServiceActionFailure(context, serviceInstance, createFailedException);